src/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-metaTitle: AssemblyScript
-description: Definitely not a TypeScript to WebAssembly compiler 🚀
+metaTitle: The AssemblyScript Book
```

Comment:
This is the start page title, so perhaps just AssemblyScript makes sense here?

Reply:
Sure